tailwind.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
You added this file for a reason, if I may ask, why?

It is weird to me to see this file because app initialized with create-react-app won't generate this.

Keep in mind that, this project required making use of the `create-react-app` script

Reply:
Hello Bellom, I added that because I want to use Tailwind for my UI. I followed Tailwind instructions here: https://tailwindcss.com/docs/guides/create-react-app 
The application was made with create-react-app.